blocks/inline-blocks/inline-image.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
+	id: 'inline-image',
+	title: __( 'Inline Image' ),
+	type: 'image',
+	icon: 'format-image',
+	render( { url, alt, width }, editor ) {
+		const imgWidth = width > 150 ? 150 : width;
+		const img = `<img style="width:${ imgWidth }px;" src="${ url }" alt="${ alt }" />`;
```

Comment:
1. Oh now that I think of it, it would be good to include the ID class name here like we do in the image block so images are caught by the WordPress' display filter and made responsive. (You can check on the front-end to test this).

2. Why setting `style` instead of `width` and `height` attributes? Does this need a comment if there's a reason?

Reply:
Good points! This commit uses your suggestions: https://github.com/WordPress/gutenberg/pull/5794/commits/c928207a243c36fe33a1e6e86272518445229f39.